hs-clj-packages.el

```diff
@@ -112,11 +112,11 @@ cider."
 (defvar hs--latest-stable-clojure-env
   '(;; Clojure Interactive Development Environment that Rocks
     (:name cider
-           :checkout "v0.14.0"
+           :checkout "v0.17.0"
            :after (progn (load-cider-config)))
     ;; A collection of simple clojure refactoring functions
     (:name clj-refactor
-           :checkout "2.2.0"
+           :checkout "2.3.1"
            :after (progn (load-clj-refactor-config))))
```

Comment:
same here. Let's wait a few days for the 2.4.0 release. Would prefer to merge this change in with that. If it doesn't get created upstream by then, then lets drop to 2.3.1.

Also, ideally the version of this should always match the corresponding version of refactor-nrepl